File: Learning/Reverse.cs
using System;

namespace C_ {
    class Reverse {
        static void Main(string[] args) {
            int reverse;
            Console.Write("Enter your number: ");
            var number = int.Parse(Console.ReadLine());
            do {
                reverse = number % 10;
                Console.Write(reverse);
                number /= 10;
            }
             while (number != 0);
             
            Console.WriteLine();
        }
    }
}

File: Introduction/reverse.c
#include <stdio.h>
/**
 * @c program to reverse numbers.
 * @return right to left
 */
int main(void)
{
    int numbers, reverse;
    printf("Enter your number: ");
    scanf("%i", &numbers);

    do {
        reverse = numbers % 10;
        printf("%i", reverse);
        numbers /= 10;
    }
    while (numbers > 0);
    
    puts("");
}